Make.versions

```diff
@@ -0,0 +1,47 @@
+#
+# A release branch requires updating the following two variables at the bottom of this file:
+#
+# IOS_PACKAGE_VERSION (major/minor/revision #)
+# MAC_PACKAGE_VERSION (major/minor/revision #)
+#
+# Update version numbers on master as well, to the next version
+#
+
+#
+# ** Version numbers **
+#
+# Versions are defined as: Major.Minor.Revison.Build
+#
+# Major/minor (first/second numbers - max 2 digits each):
+# - Bump for major/minor releases.
+#
+# Revision (third number - max 2 digits):
+# - Reset to 0 after a major or minor bump (do not use 99 for Xcode preview
+#   branches (use 0 instead), because otherwise we can't bump it further if
+#   needed).
+# - Bump for service releases and previews.
+# - Bump if commit distance becomes > 999.
+# - Can also be bumped for other reasons (in particular there's no correlation
+#   between Preview/Service Release #X and Revision #Y).
+#   - Bumping revision to a high enough number to make it clear that there's
```

Comment:
I don't understand this one. Past examples?

Reply:
we use a revision of `99` for Xcode previews

Reply:
But you said above we shouldn't do that? (:

Reply:
It's generally a **minor** `99` (for major Xcode) or an odd minor number (for minor Xcode).
But there were cases where we're stuck in _odd_ positions and used `99` in revisions (e.g. when the next _even_ number was already used)- but it's not common and there are alternatives (so many numbers)